app/controllers/pages_controller.rb

```diff
@@ -2,12 +2,13 @@ class PagesController < ApplicationController
   skip_before_action :authenticate_user!, only: [:home]
 
   def home
-<<<<<<< HEAD
+
       @properties = Property.all
```

Comment:
can you delete the property.all? claire told me we should only keep the first 3 thing

Reply:
cause I actually already change it correctly in my previous push 2 min ago so you didn't need to do it haha

Reply:
Okay